code/drasil-lang/Language/Drasil/Expr/Display.hs

```diff
@@ -0,0 +1,32 @@
+{-# LANGUAGE GADTs #-}
+
+module Language.Drasil.Expr.Display (DisplayExpr(..), Display(..),
+  defines, multiExpr, multiExprNE) where
+
+import qualified Data.List.NonEmpty as NE
+
+import Language.Drasil.Expr (Expr)
+
+data DisplayExpr where
+    AlgebraicExpr :: Expr -> DisplayExpr
+    Defines       :: DisplayExpr -> DisplayExpr -> DisplayExpr
+    MultiExpr     :: NE.NonEmpty DisplayExpr -> DisplayExpr
```

Comment:
AlgebraicExpr is just the basic expressions. I leave it without UIDs (yet?) because I'm not yet sure of the impacts. This is also why the `Defines`'s LHS is a DisplayExpr as well (as opposed to a UID).

Reply:
I think this definition is good as is. It really ought to be an embedding of `Expr` (your first constructor) along with some extras that are there just for display purposes (the rest).

Reply:
Thank you!